pkg/products/monitoring/reconciler_test.go

```diff
@@ -376,7 +376,9 @@ func TestReconciler_fullReconcile(t *testing.T) {
 		{
 			Name:           "test successful reconcile",
 			ExpectedStatus: integreatlyv1alpha1.PhaseCompleted,
-			FakeClient:     moqclient.NewSigsClientMoqWithScheme(scheme, ns, operatorNS, federationNs, grafanadatasourcesecret, installation, smtpSecret, pagerdutySecret, dmsSecret, alertmanagerRoute),
+			FakeClient: moqclient.NewSigsClientMoqWithScheme(scheme, ns, operatorNS, federationNs,
+				grafanadatasourcesecret, installation, smtpSecret, pagerdutySecret,
+				dmsSecret, alertmanagerRoute),
```

Comment:
Interested in your opinion between that style of wrapping versus the following:

```suggestion
			FakeClient: moqclient.NewSigsClientMoqWithScheme(
				scheme,
				ns,
				operatorNS,
				federationNs,
				grafanadatasourcesecret,
				installation,
				smtpSecret,
				pagerdutySecret,
				dmsSecret,
				alertmanagerRoute,
			),
```

Reply:
This style in my opinion refers to instantiation part (creating an object with initial parameters) rather than a function call for some people (may be most people :)) . What if the method takes another 10 parameters, then it would be scrolling a page vertically to see the parameters (the horizontal problem -- shifted vertically).

Reply:
If the method expected another 10 parameters, I'd maybe suggest we have a different problem (it's already borderline absurd) :slightly_smiling_face: 

I'm happy with either option (or allowing the line to run long _where it might make sense_). :+1: